build.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:convert';
+import 'dart:io';
+
+void main() async {
+  var baseConfigFile = File('config/base_config.json');
+  var projConfigFile = File('config/project_config.json');
+  if (!await baseConfigFile.exists() || !await projConfigFile.exists()) {
+    throw ArgumentError('Config files not present');
+  }
+
+  var baseConfigRaw = await baseConfigFile.readAsString();
+  Map<String, dynamic> baseConfig = jsonDecode(baseConfigRaw);
+  var projConfigRaw = await projConfigFile.readAsString();
+  Map<String, dynamic> projConfig = jsonDecode(projConfigRaw);
+
+  var buildStatus = projConfig.map((proj, projConfig) {
+    print('---------- ${proj} ----------');
+    print('${proj}: Building project ...');
+    var newConfig = {...baseConfig, ...projConfig};
+    var newFile = File('web/assets/firebase-constants.json');
```

Comment:
it might make sense to rename this file from `firebase_constants.json` to just `constants.json` or `platform_constants.json` or something else, as it's currently not storing only firebase constants, but also mapbox constants

Reply:
Makes sense, renamed to `constants.json`